### ISSUE TITLE
Use of instance parameter results in wrong settings file being converted

### ISSUE THREAD

**Issue by SL-Gundam** [ShokoAnime/ShokoServer#806]
﻿## VERSION INFORMATION

Server Version: 3.9.5

Desktop Version: 3.9.5

Tried the latest daily on 13-10-2019 05:48 (gmt+2)

## LOG FILE

n/a

## DESCRIPTION

I'm running two instances of Shoko. Since upgrading from 3.9.4 to 3.9.5 the instance parameter does not work properly anymore.

ShokoServer.exe instance ShokoServer
ShokoServer.exe instance ShokoServer2

Some further testing resulted in the instance parameter working but Shoko only loading the database from the default instance

**Comment by da3dsoul** [comment]
3.9.5 introduced the new settings iirc. I'll look at it

**Comment by da3dsoul** [comment]
What database type? CLI or UI?

**Comment by SL-Gundam** [comment]
Windows 10 1903 x64
MySQL/MariaDB on a remote NAS system

Does that give you enough information or did i misunderstand the question?

**Comment by da3dsoul** [comment]
Good enough. I assume that each instance uses a different DB?

**Comment by SL-Gundam** [comment]
Of course. Thats how i noticed that the instance worked partially (everything except the correct database)

But thanks to some hints in the changelog I've found the issue. Apparently the settings file format has changed. During the start of another instance the old settings file from the default instance is loaded and saved to the secondary instance as the new settings file.

So basically it loads the wrong settings file for conversion.

For me I've manually edited the new settings file as necessary and now everything works as required
